libs/ui/src/lib/input/input.component.ts

```diff
@@ -15,6 +15,21 @@ export const inputComponentValueAccessor = {
   multi: true,
 };
 
+type InputType = '' | 'positiveNumbers' | 'estonianNameLetters';
+
+const InputRegex: { [key: string]: RegExp }= {
+  positiveNumbers: /\D/g,
+  estonianNameLetters: /[^A-Za-zÄÜÖÕäüöõ\s-]+/
```

Comment:
Would it make sense to expand this to at least Extended Latin? Are we sure that characters like `ñ`, `ž`, `š`, `č` will never be used in a person's name on the Estonian gov portal?

It also probably makes sense to change the property name from `estonianNameLetters` to something else (remove the "name" part – it makes possible usage of this validation unnecessarily limited).

Reply:
I wont be using the estonianNameLetters option(at least not now) in the Estonian gov portal. I put it there for demo purposes, to show how this function can be used in the future. I will remove the estonianNameLetter option from there.